LandSandBoat/server
# [Chore] Create job trait enum and use when appropiate

**_I affirm:_**

## What does this pull request do?

As title sais.

## Steps to test these changes

Play regularly. Use a high lvl THF with ambush, assassin, etc and WS. Or use rampart with Iron will trait.
No changes should be observed.
